context/src/main/java/io/grpc/Context.java

```diff
@@ -719,8 +719,9 @@ public void detach(Context toAttach) {
       uncancellableSurrogate.detach(toAttach);
     }
 
+    // Visible for testing
     @Override
-    public boolean isCurrent() {
+    boolean isCurrent() {
```

Comment:
The thing about that is that it is public, stable API.  

Even if we could change it, it would be a sign of badwill towards our users.  I vote for leaving it alone.